tests/sentry/middleware/integrations/parsers/test_vsts.py

```diff
@@ -56,8 +56,9 @@ def test_routing_properly(self):
         ) as get_response_from_control_silo, mock.patch.object(
             parser, "get_regions_from_organizations", return_value=[]
         ):
-            parser.get_response()
-            assert get_response_from_control_silo.called
+            response = parser.get_response()
+            assert not get_response_from_control_silo.called
+            assert response.status_code == 202
```

Comment:
This test is a little weird, in that it mocks out the get region call. Theoretically this code path shouldn't exist.

Reply:
Yeah that makes sense. I'll remove it.